.gitignore

```diff
@@ -30,5 +30,5 @@ src/svtk/svtk/utils/helpers.cpython-37m-darwin.so
 src/svtk/svtk/utils/s3bam.pyc
 src/svtk/svtk/utils/utils.pyc
 src/svtk/svtk/vcfcluster.pyc
-test/
-inputs/
+/test/
+/inputs/
```

Comment:
Was this causing problems?

Reply:
Not having the leading slash was making git ignore files in subdirectories named `test` or `inputs`. Specifically I ran into it ignoring changes made to the `build_default_inputs.sh` script in `scripts/inputs`.